ledfx/effects/audio.py

```diff
@@ -74,16 +75,22 @@ def default_device_index():
             _LOGGER.warn("No audio output devices found.")
         else:
             # target device should be the name of the default_output device plus " [Loopback]"
-            target_device = (
-                f"{device_list[default_output_device]['name']} [Loopback]"
-            )
+            target_device = device_list[default_output_device]["name"]
+
```

Comment:
Should we call this default_output_device_name rather than target_device

Reply:
Will clean up